druid/src/win_handler.rs

```diff
@@ -319,23 +319,25 @@ impl<T: Data> Inner<T> {
         }
     }
 
-    /// Returns `true` if the command was handled.
-    fn dispatch_cmd(&mut self, cmd: Command) -> bool {
-        if !self.delegate_cmd(&cmd) {
-            self.do_update();
-            return true;
+    fn dispatch_cmd(&mut self, cmd: Command) -> Handled {
+        let handled = self.delegate_cmd(&cmd);
+        // We do the update whether or not the command was handled, because the delegate has
+        // arbitrary user code in it: it could change the data but return Handled::No.
```

Comment:
This comment seems a bit irritating to me, outside the context of this diff.
At first, I wanted to suggest a different phrasing, but I think removing this one would be best.
The code is quite clear about running `do_update` after a call to `delegate_cmd` and afterwards to an early return if the delegate handled the event.